src/components/Products/ProductsFilter.js

```diff
@@ -0,0 +1,289 @@
+
+                        <label htmlFor='size-9' className="txt-cap">9</label>
+
+                    </div>
+
+                    <div className='flx flx-min-center'>
+
+                        <input type='checkbox' id='size-8' name="size-group" value='8' checked={isInputIncluded(8, filterState.sizeChecks)} onChange={(e) => handleSizeCheck(e)} className='mg-right-xs' />
+
+                        <label htmlFor='size-8' className="txt-cap">8</label>
+
+                    </div>
+
+                    <div className='flx flx-min-center'>
+
+                        <input type='checkbox' id='size-7' name="size-group" value='7' checked={isInputIncluded(7, filterState.sizeChecks)} onChange={(e) => handleSizeCheck(e)} className='mg-right-xs' />
+
+                        <label htmlFor='size-7' className="txt-cap">7</label>
+
+                    </div>
+
+                </div>
+
+            </div>
+
+
+            <div className='mg-btm-lg'>
+
+                <Text classes='txt-md txt-cap mg-btm-xs'>rating</Text>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='checkbox' id='5-atleast' name="rating-group" value='5' checked={isInputIncluded(5, filterState.ratingChecks)} onChange={(e) => handleRatingCheck(e)} className='mg-right-xs' />
+
+                    <label htmlFor='5-atleast' className="txt-cap">5 stars</label>
+
+                </div>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='checkbox' id='4-atleast' name="rating-group" value='4' checked={isInputIncluded(4, filterState.ratingChecks)} onChange={(e) => handleRatingCheck(e)} className='mg-right-xs' />
+
+                    <label htmlFor='4-atleast' className="txt-cap">4 stars</label>
+
+                </div>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='checkbox' id='3-atleast' name="rating-group" value='3' checked={isInputIncluded(3, filterState.ratingChecks)} onChange={(e) => handleRatingCheck(e)} className='mg-right-xs' />
+
+                    <label htmlFor='3-atleast' className="txt-cap">3 stars</label>
+
+                </div>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='checkbox' id='2-atleast' name="rating-group" value='2' checked={isInputIncluded(2, filterState.ratingChecks)} onChange={(e) => handleRatingCheck(e)} className='mg-right-xs' />
+
+                    <label htmlFor='2-atleast' className="txt-cap">2 stars</label>
+
+                </div>
+
+                <div className='flx flx-min-center'>
+
+                    <input type='checkbox' id='1-atleast' name="rating-group" value='1' checked={isInputIncluded(1, filterState.ratingChecks)} onChange={(e) => handleRatingCheck(e)} className='mg-right-xs' />
+
+                    <label htmlFor='1-atleast' className="txt-cap">1 star</label>
+
+                </div>
```

Comment:
Even this can be extracted. You can put those values in an array and map them

Reply:
done.